packages/strapi-admin/admin/src/containers/Admin/Logout/index.js

```diff
@@ -31,9 +31,10 @@ const Logout = ({ history: { push } }) => {
   const toggle = () => setIsOpen(prev => !prev);
 
   const userInfo = auth.getUserInfo();
-  const displayName = userInfo
-    ? `${userInfo.firstname} ${userInfo.lastname}`
-    : get(userInfo, ['username'], '');
+  const displayName =
+    userInfo.firstname && userInfo.lastname
```

Comment:
need to keep the check for userInfo being defined

Reply:
Nice catch, fixed